src/actions/schema.js

```diff
@@ -36,6 +37,7 @@ function* loadUiSchema(action) {
           filter: { targetEntityType: entityTypeId, bundle, mode: 'default' },
         },
       }),
+      api('field_storage'),
```

Comment:
May I ask one question: It would be nice to just load the field storage entries we need. it should be possible to restrict the fields per entity type and bundle